### DISCUSSION TITLE
How to add a middleware to backpack dashboard?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Backpack/community-forum#861] [Q&A (Help)] [answered]
I added a middleware to ALL routes in the groups in the `custom.php` route file.

But it's not included the dashboard. 
How can I add a middleware to the dashboard?

**Comment by pxpm** [comment, marked as answer]
Hey @realtebo 

Does adding your middleware in `config/backpack/base.php`, in **middleware_class** key solve your issue ? 

As long as you add the `backpack_middleware()` key on the logged in routes where you want to add that middleware too it should work.

Let me know,

Cheers

**Reply by realtebo** [reply]
Thanks, worked ! I removed the middleware from routes file and added only in the base.php

**Comment by jcastroa87** [comment]
Close as solve. If you need any further information, feel free to reopen or create a new one.

Cheers.
